week10_planning/seminar_MCTS.ipynb

```diff
@@ -6,46 +6,58 @@
    "source": [
     "## Seminar: Monte-carlo tree search (5 pts)\n",
     "\n",
-    "In this seminar, we'll implement a vanilla MCTS planning and use it to solve some Gym envs.\n",
+    "Monte Carlo tree search (MCTS) is a heuristic search algorithm, which shows cool results in challenging domains such as Go and chess. The algorithm involves building a search tree, iteratively traversion  and evaluation the nodes with Monte-Carlo simulation.\n",
```

Comment:
This does not read quite right:

> The algorithm involves building a search tree, iteratively traversion  and evaluation the nodes with Monte-Carlo simulation.

How about this?

> The algorithm builds a search tree, iteratively traverses it, and evaluates its nodes using a Monte-Carlo simulation.

Reply:
Your option is better. Thanks.